numba_dpex/dpctl_iface/libsyclinterface_bindings.py

```diff
@@ -71,7 +71,7 @@ def dpctl_event_wait(builder: llvmir.IRBuilder, *args):
     mod = builder.module
     fn = _build_dpctl_function(
         llvm_module=mod,
-        return_ty=cgutils.voidptr_t,
+        return_ty=llvmir.VoidType(),
```

Comment:
These are unrelated changes. I have also made the same changes in https://github.com/IntelPython/numba-dpex/pull/1178. 

The changes should be in a separate commit no need to squash into the SyclEvent changes commit